src/optimizer/plan_nodes/logical_aggregate.rs

```diff
@@ -69,13 +69,25 @@ impl_plan_tree_node_for_unary!(LogicalAggregate);
 impl PlanNode for LogicalAggregate {
     fn schema(&self) -> Vec<ColumnDesc> {
         let child_schema = self.child.schema();
+        let mut out_names = vec![];
         let mut input_refs = vec![];
-        self.group_keys
-            .iter()
-            .for_each(|expr| expr.resolve_input_ref(&mut input_refs));
+        self.group_keys.iter().for_each(|expr| {
```

Comment:
I don't think we need to have `input_refs` vec here. Just map group_keys to ColumnDesc, and chain it will agg_call is enough.

Reply:
The `resolve_input_ref` is useful in somewhere else, but not here 🤣 I forgot why we write such thing here.